sdks/universal-router-sdk/test/forge/MigratorCallParameters.t.sol

```diff
@@ -85,8 +114,33 @@ contract MigratorCallParametersTest is Test, Interop, DeployRouter {
         assertEq(v4PositionManager.balanceOf(RECIPIENT), 1, "V4 NOT MINTED");
     }
 
-    function test_migrate_withPermitAndPoolInitialize() public {
-        MethodParameters memory params = readFixture(json, "._MIGRATE_WITH_PERMIT_AND_POOL_INITIALIZE");
+    function test_migrate_toErc20_withPermit() public {
+        MethodParameters memory params = readFixture(json, "._MIGRATE_TO_ERC20_WITH_PERMIT");
+
+        // add the position to v3 so we have something to migrate
+        assertEq(INonfungiblePositionManager(V3_POSITION_MANAGER).balanceOf(from), 0);
+        // USDC < WETH
+        mintV3Position(address(USDC), address(WETH), 3000, 2500e6, 1e18);
+        assertEq(INonfungiblePositionManager(V3_POSITION_MANAGER).balanceOf(from), 1);
+
+        assertEq(params.value, 0);
+        vm.prank(from);
+        (bool success,) = address(router).call(params.data);
+        require(success, "call failed");
+
+        // all funds were swept out of contracts
+        assertEq(USDC.balanceOf(MAINNET_ROUTER), 0);
+        assertEq(WETH.balanceOf(MAINNET_ROUTER), 0);
+        assertEq(USDC.balanceOf(address(v4PositionManager)), 0);
+        assertEq(WETH.balanceOf(address(v4PositionManager)), 0);
+
+        // old position burned, new position minted
+        assertEq(INonfungiblePositionManager(V3_POSITION_MANAGER).balanceOf(from), 0, "V3 NOT BURNT");
+        assertEq(v4PositionManager.balanceOf(RECIPIENT), 1, "V4 NOT MINTED");
```

Comment:
can you check the ETH balance of v4 didnt change, and the WETH balance did? to show it migrated to WETH not ETH?